pkg/model/crd.go

```diff
@@ -629,15 +629,17 @@ func (r *CRD) GoCodeSetInput(
 		var found bool
 		sourceAdaptedVarName := sourceVarName
 		crdField, found = r.SpecFields[renamedName]
+		// TODO(muvaf): The field path here for the actual spec fields should be
+		// configurable.
 		if found {
-			sourceAdaptedVarName += ".Spec"
+			sourceAdaptedVarName += ".Spec.ForProvider"
```

Comment:
In Crossplane, we always keep cloud provider related things under either `spec.forProvider` and `status.atProvider`. So, it'd be awesome if we can configure the this string.